gaffer-as-a-service/gaas-ui/src/components/schema-builder/add-type.tsx

```diff
@@ -91,6 +91,8 @@ export default function AddType(props: IProps): ReactElement {
     };
 
     function addTypeReducer(draft: any, action: any) {
+        const typeName = new RegExp("^[a-zA-Z]+$");
```

Comment:
Copyright header is missing, I expect this is the case for various other files as well.

Reply:
added missing copyright headers